A small patch found among Ubuntu and Debian ports of Ekiga.

	-mi

--- lib/gmconf/gmconf-glib.c	2007-09-18 09:00:21.000000000 +0000
+++ lib/gmconf/gmconf-glib.c	2007-10-06 18:34:01.000000000 +0000
@@ -1473,7 +1473,7 @@
 }
 
 gboolean 
-gm_conf_is_key_writable (gchar *key)
+gm_conf_is_key_writable (const gchar *key)
 {
   g_return_val_if_fail (key != NULL, FALSE);
 
